server/etcdserver/corrupt.go

```diff
@@ -477,8 +477,10 @@ func (s *EtcdServer) getPeerHashKVs(rev int64) []*peerHashKVResp {
 		respsLen := len(resps)
 		var lastErr error
 		for _, ep := range p.eps {
+			var resp *pb.HashKVResponse
+
 			ctx, cancel := context.WithTimeout(context.Background(), s.Cfg.ReqTimeout())
-			resp, lastErr := HashByRev(ctx, s.cluster.ID(), cc, ep, rev)
+			resp, lastErr = HashByRev(ctx, s.cluster.ID(), cc, ep, rev)
```

Comment:
create new var to prevent shadow the `lastErr`.

etcdserver/corrupt.go:481:10: declaration of "lastErr" shadows declaration at line 478